<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchTMDialog</class>
 <widget class="QDialog" name="BatchTMDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch Tone Mapping</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/new/prefix1/images/luminance.png</normaloff>:/new/prefix1/images/luminance.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="_2">
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>9</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>HDR Images to Convert</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <layout class="QHBoxLayout" name="_6">
        <property name="margin">
         <number>2</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="_3">
          <item>
           <widget class="QPushButton" name="add_dir_HDRs_Button">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Add all the HDRs in a directory to the list</string>
            </property>
            <property name="text">
             <string>Add &amp;Directory</string>
            </property>
            <property name="icon">
             <iconset resource="../../icons.qrc">
              <normaloff>:/new/prefix1/images/fileopen.png</normaloff>:/new/prefix1/images/fileopen.png</iconset>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="add_HDRs_Button">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Add single HDR files to the list</string>
            </property>
            <property name="text">
             <string>&amp;Add Files</string>
            </property>
            <property name="icon">
             <iconset resource="../../icons.qrc">
              <normaloff>:/new/prefix1/images/add.png</normaloff>:/new/prefix1/images/add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="remove_HDRs_Button">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Remove single HDR files to the list</string>
            </property>
            <property name="text">
             <string>&amp;Remove Files</string>
            </property>
            <property name="icon">
             <iconset resource="../../icons.qrc">
              <normaloff>:/new/prefix1/images/remove.png</normaloff>:/new/prefix1/images/remove.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget_HDRs">
          <property name="toolTip">
           <string>List of HDRs that will be tone mapped</string>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ContiguousSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Tone Mapping Settings</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <layout class="QHBoxLayout" name="_5">
        <property name="margin">
         <number>2</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="_4">
          <item>
           <widget class="QPushButton" name="add_dir_TMopts_Button">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Add all the Tone Mapping Setting files in a directory to the list</string>
            </property>
            <property name="text">
             <string>Add D&amp;irectory</string>
            </property>
            <property name="icon">
             <iconset resource="../../icons.qrc">
              <normaloff>:/new/prefix1/images/fileopen.png</normaloff>:/new/prefix1/images/fileopen.png</iconset>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="add_TMopts_Button">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Add single Tone Mapping Setting files to the list</string>
            </property>
            <property name="text">
             <string>Add Fi&amp;les</string>
            </property>
            <property name="icon">
             <iconset resource="../../icons.qrc">
              <normaloff>:/new/prefix1/images/add.png</normaloff>:/new/prefix1/images/add.png</iconset>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="from_Database_Button">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string>From Database</string>
            </property>
            <property name="icon">
             <iconset resource="../../icons.qrc">
              <normaloff>:/new/prefix1/images/draganddrop.png</normaloff>:/new/prefix1/images/draganddrop.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="remove_TMOpts_Button">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Remove single Tone Mapping Setting files to the list</string>
            </property>
            <property name="text">
             <string>R&amp;emove</string>
            </property>
            <property name="icon">
             <iconset resource="../../icons.qrc">
              <normaloff>:/new/prefix1/images/remove.png</normaloff>:/new/prefix1/images/remove.png</iconset>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget_TMopts">
          <property name="toolTip">
           <string>List of Tone Mapping Setting files that will be used to tone map each HDR</string>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ContiguousSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="toolTip">
      <string>Specify output folder for the tone mapped files</string>
     </property>
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QHBoxLayout" name="_7">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="out_folder_Button">
        <property name="text">
         <string>Select &amp;Output Folder...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="out_folder_widgets">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Conversion Log</string>
     </property>
     <layout class="QVBoxLayout" name="_8">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QListView" name="Log_Widget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Batch operations report</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ContiguousSelection</enum>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="filterLabel1">
          <property name="text">
           <string>&amp;Show only:</string>
          </property>
          <property name="buddy">
           <cstring>filterComboBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="filterComboBox">
          <property name="toolTip">
           <string>Filter messages based on severity</string>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>All messages</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Errors only</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Success messages</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="filterLabel2">
          <property name="text">
           <string>&amp;Filter log messages:</string>
          </property>
          <property name="buddy">
           <cstring>filterLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="filterLineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="clearTextToolButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Clear filter text</string>
          </property>
          <property name="icon">
           <iconset resource="../../icons.qrc">
            <normaloff>:/new/prefix1/images/fileclose.png</normaloff>:/new/prefix1/images/fileclose.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="overallProgressBar">
     <property name="toolTip">
      <string>Overall completion progress</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelbutton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BatchGoButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Start batch tone mapping</string>
       </property>
       <property name="text">
        <string>&amp;Start</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>remove_HDRs_Button</tabstop>
  <tabstop>out_folder_Button</tabstop>
  <tabstop>filterComboBox</tabstop>
  <tabstop>cancelbutton</tabstop>
  <tabstop>BatchGoButton</tabstop>
  <tabstop>listWidget_HDRs</tabstop>
  <tabstop>out_folder_widgets</tabstop>
 </tabstops>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
